learning-bitmark/working-with-bitmarks/payment-for-bitmark-cli.md

```diff
@@ -0,0 +1,261 @@
+# Bitmark transaction payment
+
+In the Bitmark Property System, for each asset, each Bitmark Account is able to issue a bitmark for free, and has to pay for all the other transactions such as bitmark transfers or other bitmark issuances.
+
+The Bitmark CLI submits transactions to its connected node via RPC. Once receiving these transactions, the node processes them and returned the corresponding payment ids along with the payment amounts and addresses. Only when the payment - either by bitcoin or litecoin - is executed, a transaction is updated from `Pending` to `Verified` and is ready to be mined. The payment transaction needs to send:
+
+* A correct amount
+
+* To the correct address
+
+* With the correct payment id added as bitcoin/litecoin OP_RETURN opcode.
+
+Because common wallets do not support attaching the Bitmark blockchain payment identifier to the ltc/btc transactions, we provide an app called the Bitmark Wallet to pay for the Bitmark transactions come from the Bitmark CLI.
+
+<br>
+<br>
+
+## Bitmark Wallet
+
```

Comment:
Is this a command-line wallet? A graphic wallet? A little more explanation is needed, so that readers know what to expect.

Also: how do users access this wallet? Is it part of the bitmark-cli download? Is it a separate add-on that they must download.

Reply:
(Ah, I now see the installation instructions later. Probably need a mention here.)

Reply:
Updated text:
```The wallet is a command-line tool which is written in Go and cannot work standalone, it can only send the bitcoin and litecoin transactions via a correct configured bitcoind and litecoind. Depending on the configuration of bitcoind and litecoind, the payment transactions are submitted to the mainnet coins or testnet coins.```